opencraft/settings.py

```diff
@@ -764,6 +764,9 @@
 # The basic auth password needed to access the node exporter.
 NODE_EXPORTER_PASSWORD = env('NODE_EXPORTER_PASSWORD', default=None)
 
+# Thresholds for NRQL alert conditions
+NEWRELIC_NRQL_ALERT_CONDITION_DURATION = env('NEWRELIC_NRQL_ALERT_CONDITION_DURATION', default='16')
```

Comment:
Nice idea of setting the threshold using a setting, we can easily adjust it after some testing.